crates/handler/src/frame.rs

```diff
@@ -316,10 +316,12 @@ where
         let caller_balance = context
             .journal()
             .load_account(inputs.caller)?
-            .map(|a| a.info.balance);
+            .data
+            .info
+            .balance;
```

Comment:
hmm, this went from `info.balance.data` to `data.info.balance`?

or am I missing something

Reply:
This is fine, `.map` is from `StateLoad` struck that maps data to a different type.